doc/multisig-tutorial.md

```diff
@@ -0,0 +1,242 @@
+# 1. Multisig Tutorial
+
+Currently, it is possible to create a multisig wallet using Bitcoin Core only.
+
+Although there is already a brief explanation and a functional test about the multisig implemented in [PR 22067](https://github.com/bitcoin/bitcoin/pull/22067), this tutorial proposes to use the signet (instead of regtest), bringing the reader closer to a real environment and explaining some functions in more detail.
```

Comment:
Not sure if it makes sense to link to a pull request on GitHub for documentation, given that it is already merged.

Reply:
Ah yes, that should link to https://github.com/bitcoin/bitcoin/blob/master/doc/descriptors.md#multisig now

Reply:
Done in 1b58755